composer.json

```diff
@@ -81,7 +81,7 @@
 			"Yoast\\WP\\SEO\\Composer\\Actions::check_coding_standards"
 		],
 		"check-cs-thresholds": [
-			"@putenv YOASTCS_THRESHOLD_ERRORS=463",
+			"@putenv YOASTCS_THRESHOLD_ERRORS=468",
```

Comment:
Oeh, this is not good. Is there a way to fix the 5 issues?

Reply:
I added a few classes that have names with more than 3 words, which PHPCS objects to.

For example:
 * `Post_Not_Found_Exception`
 * `Term_Not_Found_Exception`
 * `Elementor_Exclude_Post_Types`
 * `Exclude_Oembed_Cache_Post_Type`
And their respective tests.

I am struggling to find descriptive names that have three words or less...

We can exclude these files from the `Yoast.NamingConventions.ObjectNameDepth.MaxExceeded` sniff, but that is a bit of a crutch... Can we discuss this sniff with the team? It is one of those sniffs that I always find difficult to solve (naming things is complicated).

Reply:
I did manage to shave of two errors.